suite/runsuite_wrapper.pl

```diff
@@ -391,6 +391,7 @@
                 'prof_pcs,thread_private|common.nativeexec_bindnow_opt' => 1, # i#2052
                 );
             %ignore_failures_64 = (
+                'api.rseq' => 1, # i#6612
```

Comment:
Have you seen it fail in other than `code_api|`?  Or you're just being proactive, just in case?

Reply:
I have not, I followed our document and assumed we didn't want to include `code_api|`.
Although, `ctest -N` shows that only `code_api|api.rseq` exists, so I'll put the full name.

Reply:
Curious which document this is?

It would have been fine to not have the prefix; tradeoffs between tighter relaxation to avoid hiding problems vs not relaxing enough so could see it either way.

Reply:
The "bespoke circuits weekly" it states only `api.rseq` without `code_api|`.

Reply:
Ah, that was me using shorthand; nothing deliberately implied.

Reply:
Our internal document :)